GeosCore/convection_mod.F90

```diff
@@ -1180,7 +1167,8 @@ SUBROUTINE DO_RAS_CLOUD_CONVECTION( Input_Opt,  &
                    CALL GC_Error( ErrMsg, RC, ThisLoc )
                    RETURN
                 ENDIF
-
+            ENDIF
```

Comment:
I think the `#ifdef LUO_WETDEP` at line 1149 is redundant, as there is another `#ifdef LUO_WETDEP` at line 1125.